logging.cfg

```diff
@@ -0,0 +1,65 @@
+[logger_root]
```

Comment:
Add a comment indicating that this is a sample logging config file to be used with the `--logging-config` argument